fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -419,8 +424,11 @@ def load_password_for_transporter
     def handle_two_step_failure(ex)
       if ENV[TWO_FACTOR_ENV_VARIABLE].to_s.length > 0
         # Password provided, however we already used it
-        UI.error("Application specific password you provided using #{TWO_FACTOR_ENV_VARIABLE}")
-        UI.error("is invalid, please make sure it's correct")
+        UI.error("")
+        UI.error("Application specific password you provided using")
+        UI.error("environment variable #{TWO_FACTOR_ENV_VARIABLE}")
+        UI.error("is invalid, please make sure it's correct.")
```

Comment:
Can you remove the `.`? Its fastlane style to not add those 😉

Reply:
Its just the way things have been generally printed out in the rest of fastlane so would probably like to keep things consistent if possible 😊

Reply:
ok.